.github/workflows/triage.yml

```diff
@@ -41,7 +41,7 @@ jobs:
               --header 'Accept: application/vnd.github+json' \
               --header 'X-GitHub-Api-Version: 2022-11-28' \
               'repos/{owner}/{repo}/pulls/${{ github.event.number }}/files' \
-              --jq '.[].filename | map(startswith(".github/workflows")) | any'
```

Comment:
Oops, the version I thought I pushed was
```jq
[.[].filename] | map(startswith(".github/workflows")) | any
```
which would have worked. Though I think dropping `map` may also have worked here.